Userland/Applications/Browser/StorageWidget.cpp

```diff
@@ -44,4 +58,14 @@ void StorageWidget::clear_cookies()
     m_cookies_model->clear_items();
 }
 
+void StorageWidget::set_local_storage_objects(OrderedHashMap<String, String> objects)
+{
+
+    m_local_storage_model->set_items(objects);
+}
+
+void StorageWidget::clear_local_storage_objects()
+{
+    m_local_storage_model->clear_items();
+}
```

Comment:
Please separate the closing namespace curly bracket with a newline

Reply:
Surprised `clang-format` didn't catch that!

Reply:
Yeah, it's a bit limited in that regard. Someone should probably file a bug! :^)